chapters/06-summary.Rmd

```diff
@@ -1,15 +1,16 @@
 # Summary {#summary}
 
-The benchmarking has shown that the new implementation can outperform the de-facto state of the art library 'KDEpy' in terms of runtime and accuracy for large datasets ($n \geq 10^8$).
+After discussing the current mathematical research on kernel density estimation, several new implementations for kernel density estimation using zfit and TensorFlow were proposed. Namely one just using linear binning (`ZfitBinned`), one using an approach based on a Fast Fourier Transform (`ZfitFFT`), one based on the improved Sheather-Jones algorithm (`ZfitISJ`) and one based on specialized kernel functions of the form $poly(x)\cdot\exp(x)$ (`ZfitHofmeyr`). 
 
-For smaller datasets it provides accuracy of the same order of magnitude and may be favorable (faster) in cases where the PDF is built once but evaluated repeatedly. 
+All proposed implementations restrict themselves to the one-dimensional case as described in section \@ref(introduction-univariate). So far only KDEpy and Statsmodels implement a multidimensional KDE in Python. Generalization to higher dimensionel kernel density estimation is feasible, although this would require more work. Firstly because the binning routine would have to be extended to the multi-dimensional case and secondly because the improved Sheather-Jones algorithm implementation would have to be adapted. In addition one must ensure that the kernel functions used are multidimensional themselves, however, this is already the case for the most important kernel function, the Gaussian density.
```

Comment:
Since this talks about future extensions, I would move this further down to the end of the summary, basically second last paragraph?